app/server/api/credit_transfer_api.py

```diff
@@ -178,6 +178,10 @@ def post(self, credit_transfer_id):
         target_balance = post_data.get('target_balance')
 
         transfer_use = post_data.get('transfer_use')
+        try:
+            use_ids = transfer_use.split(',')  # passed as '3,4' etc.
+        except AttributeError:
+            use_ids = transfer_use
 
```

Comment:
Confused as to where/how this is being used?

Reply:
ohhh i can comment here!
this was originally buried inside the create payment method - the mobile passes back transfer usages as a string. Ideally, we wouldn't have to parse strings at all, but in cases where we do, this process should be at the api level not buried in a method

Reply:
just force pushed

Reply:
I think these changes are supposed to be on #394

Reply:
Ahh, Sorry!

Reply:
All good!